ddp-workspace/projects/ddp-atcp/src/app/app.module.ts

```diff
@@ -0,0 +1,196 @@
+
+export let config = new ConfigurationService();
+config.backendUrl = DDP_ENV.basePepperUrl;
+config.auth0Domain = DDP_ENV.auth0Domain;
+config.auth0ClientId = DDP_ENV.auth0ClientId;
+config.studyGuid = DDP_ENV.studyGuid;
+config.logLevel = LogLevel.Info;
+config.baseUrl = location.origin + base;
+config.auth0SilentRenewUrl = DDP_ENV.auth0SilentRenewUrl;
+config.loginLandingUrl = DDP_ENV.loginLandingUrl;
+config.auth0CodeRedirect = location.origin + base + 'auth';
+config.localRegistrationUrl = config.backendUrl + '/pepper/v1/register';
+config.doLocalRegistration = DDP_ENV.doLocalRegistration;
+config.mapsApiKey = DDP_ENV.mapsApiKey;
+config.auth0Audience = DDP_ENV.auth0Audience;
+config.projectGAToken = DDP_ENV.projectGAToken;
+
+export function translateFactory(translate: TranslateService, injector: Injector) {
+  return () => new Promise<any>((resolve: any) => {
+    const locationInitialized = injector.get(LOCATION_INITIALIZED, Promise.resolve(null));
+    locationInitialized.then(() => {
+      const locale = 'en';
+      translate.setDefaultLang(locale);
+      translate.use(locale).subscribe(() => {
+        console.log(`Successfully initialized '${locale}' language as default.`);
+      }, err => {
+        console.error(`Problem with '${locale}' language initialization.`);
+      }, () => {
+        resolve(null);
+      });
+    });
+  });
+}
+
+export class MyHammerConfig extends HammerGestureConfig {
+  public buildHammer(element: HTMLElement): Hammer {
+    const hammer = new Hammer(element, {
+      touchAction: 'pan-y'
+    });
+    return hammer;
+  }
+}
+
+@NgModule({
+  imports: [
+    BrowserModule,
+    FormsModule,
+    CommonModule,
+    AppRoutingModule,
+    DdpModule,
+    ToolkitModule,
+    MatExpansionModule,
+    MatIconModule,
+    MatMenuModule
+  ],
+  declarations: [
+    WelcomeComponent,
+    AppComponent,
+    FooterComponent,
+    AboutUsComponent,
+    AboutInitiativeComponent,
+    HeaderComponent,
+    JoinUsComponent,
+    DataAccessComponent,
+    PasswordComponent,
+    DashBoardComponent,
+    WorkflowProgressComponent,
+    ErrorComponent,
+    StatisticsComponent,
+    ProgressBarComponent
+  ],
+  entryComponents: [
+    DashBoardComponent,
+  ],
+  providers: [
+    {
+      provide: 'ddp.config',
+      useValue: config
+    },
+    {
+      provide: 'toolkit.toolkitConfig',
+      useValue: tkCfg
+    },
+    {
+      provide: APP_INITIALIZER,
+      useFactory: translateFactory,
+      deps: [
+        TranslateService,
+        Injector
+      ],
+      multi: true
+    },
+    {
+      provide: HAMMER_GESTURE_CONFIG,
+      useClass: MyHammerConfig
+    },
+    LanguagesProvider
+  ],
+  bootstrap: [AppComponent]
+})
+export class AppModule {
+  constructor(
+    private router: Router,
+    private analytics: AnalyticsEventsService) {
+    this.router.events.subscribe(event => {
+      if (event instanceof NavigationEnd) {
+        this.analytics.emitNavigationEvent();
+      }
+    });
```

Comment:
Please, update module's constructor like in Osteo https://github.com/broadinstitute/ddp-angular/blob/develop/ddp-workspace/projects/ddp-osteo/src/app/app.module.ts#L173